server/settings.py

```diff
@@ -143,7 +143,10 @@ def env(variable, fallback_value=None):
     'apps.packages',
     'apps.privilege',
     'apps.rule_sets',
-    'apps.highlights'
+    'apps.highlights',
+    'apps.macros',
+
+    'macros.currency',
```

Comment:
It looks like we need to restart the server when there's a new macro, is that right?